tests/test_parser.py

```diff
@@ -60,17 +60,19 @@ def test_parse_catalog_specific(self, version):
         )
 
 
-@pytest.mark.parametrize("version", ["v1", "v2", "v3", "v4", "v5", "v6", "v7", "v8"])
+@pytest.mark.parametrize("version,path", [
+    ("v1", os.path.join(get_project_root(), "tests", "resources", "v1", "jaffle_shop", "manifest.json")),
+    ("v2", os.path.join(get_project_root(), "tests", "resources", "v2", "jaffle_shop", "manifest.json")),
+    ("v3", os.path.join(get_project_root(), "tests", "resources", "v3", "jaffle_shop", "manifest.json")),
+    ("v4", os.path.join(get_project_root(), "tests", "resources", "v4", "jaffle_shop", "manifest.json")),
+    ("v5", os.path.join(get_project_root(), "tests", "resources", "v5", "jaffle_shop", "manifest.json")),
+    ("v6", os.path.join(get_project_root(), "tests", "resources", "v6", "jaffle_shop", "manifest.json")),
+    ("v7", os.path.join(get_project_root(), "tests", "resources", "v7", "jaffle_shop", "manifest.json")),
+    ("v8", os.path.join(get_project_root(), "tests", "resources", "v8", "jaffle_shop", "manifest.json")),
+    ("v8", os.path.join(get_project_root(), "tests", "resources", "v8", "jaffle_shop_at_1_4_3", "manifest.json")),
```

Comment:
We test the parser with `manifest.json` generated by dbt 1.4.1 and 1.4.3 respectively.